packages/data-connect/test/unit/fetch.test.ts

```diff
@@ -74,8 +80,106 @@ describe('fetch', () => {
         null,
         null,
         null,
-        false
+        false,
+        CallerSdkTypeEnum.Base
       )
     ).to.eventually.be.rejectedWith(JSON.stringify(json));
   });
+  it('should assign different values to custom headers based on the _callerSdkType argument (_isUsingGen is false)', async () => {
+    const json = {
+      code: 200,
+      message1: 'success'
+    };
+    const fakeFetchImpl = mockFetch(json, false);
+
+    const callerSdkTypesUsed: string[] = [];
+
+    for (const callerSdkType in CallerSdkTypeEnum) {
+      if (
+        Object.prototype.hasOwnProperty.call(CallerSdkTypeEnum, callerSdkType)
```

Comment:
Would you mind putting a comment above this to mention that this is because of the eslint rule?